src/ocm/components/clusterWizard/ClusterPlatformIntegrationHint.tsx

```diff
@@ -23,7 +23,8 @@ export const ClusterPlatformIntegrationHint = ({
 
   const canIntegrateWithPlatform =
     isPlatformIntegrationSupported &&
-    !isClusterPlatformTypeVM({ platform: { type: platformType } });
+    !isClusterPlatformTypeVM({ platform: { type: platformType } }) &&
+    supportedPlatformIntegration !== undefined;
```

Comment:
`undefined` is never the value of `supportedPlatformIntegration`, so this condition is always true and unnecessary.

Reply:
true. At first is undefined until I've modified the useClusterSupportedPlatforms function